tests/src/test/scala/com/nvidia/spark/rapids/CastOpSuite.scala

```diff
@@ -692,7 +692,10 @@ object CastOpSuite {
     import session.sqlContext.implicits._
     val df = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS")
     val timestampStrings = Seq(
-      "2020-12-31T11:59:59.999",
+      "1920-12-31T11:59:59.999",
```

Comment:
can we add a few microseconds?

Reply:
@razajafri Done. This actually highlighted a bug in the original implementation where we only supported timestamps with millisecond precision.